langserve/client.py

```diff
@@ -455,6 +459,8 @@ async def astream_log(
             async with aconnect_sse(
                 self.async_client, "POST", endpoint, json=data
             ) as event_source:
+                event_source.response.raise_for_status()
```

Comment:
How do we want to handle http exceptions in the streaming endpoints? If there's a way to split this request into sync (for exceptions) and async (for streaming response) that would be ideal, so we can read the response error message

Reply:
An error event in the stream